tests/actions/TaskTest.ts

```diff
@@ -141,4 +142,33 @@ describe('actions/Task', () => {
             expect(writeSpy).toHaveBeenCalledWith(WRITE_COMMANDS.COMPLETE_TASK, expect.anything(), expect.anything());
         });
     });
+
+    describe('getFinishOnboardingTaskOnyxData', () => {
+        const parentReport: Report = LHNTestUtils.getFakeReport();
+        const taskReport: Report = {...LHNTestUtils.getFakeReport(), type: CONST.REPORT.TYPE.TASK, ownerAccountID: 1, managerID: 2, parentReportID: parentReport.reportID};
+        const reportCollectionDataSet: ReportCollectionDataSet = {
+            [`${ONYXKEYS.COLLECTION.REPORT}${taskReport.reportID}`]: taskReport,
+            [`${ONYXKEYS.COLLECTION.REPORT}${parentReport.reportID}`]: parentReport,
+        };
+        beforeEach(async () => {
+            await Onyx.clear();
+            await Onyx.multiSet({
+                ...reportCollectionDataSet,
+            });
+            await Onyx.set(ONYXKEYS.SESSION, {email: 'user1@gmail.com', accountID: 2});
+            await Onyx.set(`${ONYXKEYS.NVP_INTRO_SELECTED}`, {choice: CONST.ONBOARDING_CHOICES.MANAGE_TEAM, setupCategories: taskReport.reportID});
+            await waitForBatchedUpdates();
+        });
+        it('Return not empty object', () => {
```

Comment:
Hi, just dropping in here, but can you please be more descriptive with the test names or comments? From looking at this test (and the one below), I have no context to understand why an empty or a non-empty object would be expected.